src/components/layout/Tutorial/Step/CreateVenue.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CreateVenue = () => (
+  <div>
+    <h1>
```

Comment:
```suggestion
    <h1 id="tuto-dialog-label">
```
cf. commentaire à propos de labelledBy dans TutorialModal.jsx

Reply:
A verifier que la DialogBox gere le fai que ce composant change durant la vie de la modal.